tsnet/tsnet.go

```diff
@@ -540,6 +540,7 @@ func (ln *listener) Close() error {
 	}
 	return nil
 }
+func (ln *listener) Server() *Server { return ln.s }
```

Comment:
blank line before.

and document? well, it's exported on unexported type, so how would people use this? sniff it out of an undocumented interface? then add a test so we don't remove this in the future?

Reply:
yes, basically.  See `tsnetListener` in https://github.com/tailscale/caddy-tailscale-auth/pull/1/files.  Will add docs and test.